### PR TITLE
ADD-products-to-list

### PR DESCRIPTION
**Pull request by j-6**
add products

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -29,3 +29,4 @@ a pull request to suggest improvements or add clarifications. Please use issues
     - Exit Survey
   - Learning & Development
 - Tools
+- [Products](docs/products)
```
Comment (ghost) [inline README.md]: Considering moving Product higher in the list, (even potentially within **Company**)
- I think it is important that it is visual to all, and easy to spot

**Review by j-6 — COMMENTED** [review]
What do you think about this? Including the products makes it more visible.

**Review by ghost — APPROVED** [review]
Looks good!

**Comment by tancnle** [comment]
@talzid  @j-6 This PR has been superseded by https://github.com/thoth-tech/handbook/pull/33 and is no longer needed. Please feel free to re-open.

**Pull request closed without merging**